guides/source/active_storage_overview.md

```diff
@@ -531,68 +531,102 @@ It's important to know that the file is not yet available in the `after_create`
 Analyzing Files
 ---------------
 
-Active Storage [analyzes](https://api.rubyonrails.org/classes/ActiveStorage/Blob/Analyzable.html#method-i-analyze) files once they've been uploaded by queuing a job in Active Job. Analyzed files will store additional information in the metadata hash, including `analyzed: true`. You can check whether a blob has been analyzed by calling [`analyzed?`][] on it.
+Active Storage [analyzes][] files once they've been uploaded by queuing a job in Active Job. Analyzed files will store additional information in the metadata hash, including `analyzed: true`. You can check whether a blob has been analyzed by calling [`analyzed?`][] on it.
 
 Image analysis provides `width` and `height` attributes. Video analysis provides these, as well as `duration`, `angle`, and `display_aspect_ratio`.
 
 Analysis requires the `mini_magick` gem. Video analysis also requires the [FFmpeg](https://www.ffmpeg.org/) library, which you must include separately.
 
+[analyzes]: https://api.rubyonrails.org/classes/ActiveStorage/Blob/Analyzable.html#method-i-analyze
 [`analyzed?`]: https://api.rubyonrails.org/classes/ActiveStorage/Blob/Analyzable.html#method-i-analyzed-3F
 
-Transforming Images
--------------------
+Displaying Images, Videos, and PDFs
+---------------
+
+Active Storage supports representing a variety of files. You can call
+[`representation`][] on an attachment to display an image variant, or a
+preview of a video or PDF. Before calling `representation`, check if the
+attachment can be represented by calling [`representable?`]. Some file formats
+can't be previewed by ActiveStorage out of the box (eg. Word documents); if
+`representable?` returns false you may want to [link to](#linking-to-files)
+the file instead.
+
+```erb
+<ul>
+  <% @message.files.each do |file| %>
+    <li>
+      <% if file.representable? %>
+        <%= image_tag file.representation(resize_to_limit: [100, 100]) %>
+      <% else %>
+        <%= link_to url_for(file) do %>
+          <%= image_tag "placeholder.png", alt: "Download file" %>
+        <% end %>
```

Comment:
I don't think `link_to` is necessary in this example, but, if it is included, I think it makes more sense to put the entire `if`/`else` in the `link_to` block.

Reply:
The idea is to show that if you can't represent a file, then maybe it makes sense to link to it (for download) instead. I'll make it more explicit.

Reply:
I understand.  I was pointing out that, *as an introductory example*, it could be simplified.  Also, I feel like if you *are* linking to files, you would want to link in either case.